zkevm-circuits/src/tx_circuit.rs

```diff
@@ -3444,7 +3740,22 @@ impl<F: Field> TxCircuit<F> {
                     }
                 };
 
+                let mut chunk_bytes: Vec<u8> = vec![];
+                for i in 0..sigs.len() {
+                    let tx = get_tx(i);
+                    if (tx.tx_type != TxType::L1Msg) && !tx.caller_address.is_zero() {
+                        chunk_bytes.extend_from_slice(&tx.rlp_signed);
+                    }
+                }
+
+                let chunk_txbytes_hash = keccak256(chunk_bytes.as_slice());
+                let evm_word = challenges.evm_word();
+                let chunk_txbytes_hash = rlc_be_bytes(&chunk_txbytes_hash, evm_word);
+
                 let mut tx_value_cells = vec![];
+                let mut chunk_txbytes_rlc_acc = Value::known(F::zero());
+                let mut chunk_txbytes_len_acc = Value::known(F::zero());
+                let mut pows_of_rand: Vec<Value<F>> = vec![Value::known(F::one())];
```

Comment:
is this ok? `let mut pows_of_rand: Vec<Value<F>> = vec![Value::known(F::one())];`

Reply:
yes, as the current max real l2 tx bytes' len is 0.